build.xml

```diff
@@ -71,7 +76,7 @@
           resultproperty="test.result"
           failonerror="true">
       <arg value="-c"/>
-      <arg value=". bin/activate;cd ../extension;cfx test ${cfx.binary}"/>
+      <arg value=". bin/activate;cd ../extension;cfx --verbose test ${cfx.binary}"/>
```

Comment:
I have no idea how to write a nice paragraph about it. :P 
So feel free to provide one. I'll do the merge later tonight.